test-e2e/manager-archive-device.js

```diff
@@ -0,0 +1,140 @@
+import test from 'node:test'
+import { KeyManager } from '@mapeo/crypto'
+import { MapeoManager as MapeoManagerPreMigration } from '@comapeo/core2.0.1'
+import RAM from 'random-access-memory'
+import { MapeoManager } from '../src/mapeo-manager.js'
+import Fastify from 'fastify'
+import assert from 'node:assert/strict'
+import { kIsArchiveDevice } from '../src/mapeo-project.js'
+import { ManagerCustodian } from './utils.js'
+import { fileURLToPath } from 'node:url'
+import fs from 'node:fs'
+import { temporaryDirectory } from 'tempy'
+
+const projectMigrationsFolder = new URL('../drizzle/project', import.meta.url)
+  .pathname
+const clientMigrationsFolder = new URL('../drizzle/client', import.meta.url)
+  .pathname
+
+test('Set & Get isArchiveDevice', async () => {
+  const manager = new MapeoManager({
+    rootKey: KeyManager.generateRootKey(),
+    projectMigrationsFolder,
+    clientMigrationsFolder,
+    dbFolder: ':memory:',
+    coreStorage: () => new RAM(),
+    fastify: Fastify(),
+  })
```

Comment:
Why not use the existing test helpers for creating a manager?

Reply:
Addressed in 2dc69a6a0351cb1db0ff65b813e1b0efdccaccae